examples/onebrc/README.md

```diff
@@ -0,0 +1,216 @@
+
+5. Build the generator for the test data 
+```bash
+cd /demo/1brc
+./mvnw verify 
+```
+
+6. Use the generator to create 1 billion rows of test data
+```bash
+time ./create_measurements.sh 1000000000
+```
+
+<details>
+<summary>The generator generated <code>measurements.txt</code> (a 13GB CSV file) in about 10 minutes.</summary>
+<pre>
+time ./create_measurements.sh 1000000000
+Wrote 50,000,000 measurements in 15007 ms
+Wrote 100,000,000 measurements in 46490 ms
+Wrote 150,000,000 measurements in 77952 ms
+Wrote 200,000,000 measurements in 109410 ms
+Wrote 250,000,000 measurements in 140847 ms
+Wrote 300,000,000 measurements in 172254 ms
+Wrote 350,000,000 measurements in 203742 ms
+Wrote 400,000,000 measurements in 235155 ms
+Wrote 450,000,000 measurements in 266711 ms
+Wrote 500,000,000 measurements in 298319 ms
+Wrote 550,000,000 measurements in 329901 ms
+Wrote 600,000,000 measurements in 361338 ms
+Wrote 650,000,000 measurements in 392873 ms
+Wrote 700,000,000 measurements in 424411 ms
+Wrote 750,000,000 measurements in 455838 ms
+Wrote 800,000,000 measurements in 487368 ms
+Wrote 850,000,000 measurements in 518836 ms
+Wrote 900,000,000 measurements in 550312 ms
+Wrote 950,000,000 measurements in 582063 ms
+Created file with 1,000,000,000 measurements in 613590 ms
+
+real    10m13.660s
+user    9m54.950s
+sys     0m18.579s
+
+du -shL measurements.txt 
+13G     measurements.txt
+</pre>
+</details>
+
+
+## `1brc` Demo using Proton
+1. Install Proton on your machine:
+```bash
+cd /demo
+curl https://install.timeplus.com | sh
+```
+
+2. Start the Proton server:
+```bash
+./proton server start
+```
+
+3. Once the Proton server is started successfully, it will create a folder named `proton-data/` in the current directory which contains multiple subfolders. We will create a symbolic link to the test data we generated earlier (`measurements.txt`) from the `proton-data/user_files` subfolder created by the Proton server.
+```bash
+ln -s /demo/1brc/measurements.txt /demo/proton-data/user_files
+```
+
+4. Now put the query we will be executing against the Proton server in a file named `1brc.sql`:
+```bash
+cat <<EOF > /demo/1brc.sql
+SET format_csv_delimiter = ';';
+
+SELECT 
+    concat('{', array_string_concat(group_array(formatted_result), ', '), '}') AS final_output
+FROM (
+    SELECT 
+        format('{}={}/{}/{}', city, to_string(to_decimal(min(temperature), 1)), to_string(to_decimal(avg(temperature), 1)), to_string(to_decimal(max(temperature), 1))) AS formatted_result
+    FROM file('measurements.txt', 'CSV', 'city string, temperature float32')
+    GROUP BY city
+    ORDER BY city
+)
+EOF
+```
+
+5. Next, from another terminal, start the Proton client with the `1brc.sql` file as input:
+```bash
+time ./proton client --host 127.0.0.1 --multiquery < /demo/1brc.sql
+```
+
+<details>
+<summary>Aggregation of 1 billion rows in Proton took just <code>39.301 seconds</code> (versus <code>47.053 seconds</code> for ClickHouse) on my machine.</summary>
```

Comment:
Would you please add a summary at the end of the README/blog? Highlighting Timeplus Proton is as fast as ClickHouse, or even slightly faster.

I will run the test as you documented.

On the other hand, the current approach is to use the ClickHouse `file` function to read the file without putting them in Timeplus storage. To highlight Timeplus Proton has great unified storage for both streaming and batch, it will be better to have another section to load those 1 billion rows to Timeplus Proton, in a stream, then use `table(stream)` function to get the result and see how fast it is. With proper index/codec, I believe it will be fast too. Maybe you can find some examples using ClickHouse table engine to query the 1brc test.